recipes/fastgltf/all/conanfile.py

```diff
@@ -0,0 +1,63 @@
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout
+from conan.tools.files import get
+from conan.tools.build import check_min_cppstd
+
+class fastgltf(ConanFile):
+    name = "fastgltf"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/spnda/fastgltf"
+    description = "A blazing fast C++17 glTF 2.0 library powered by SIMD."
+    topics = ("gltf", "simd", "cpp17")
+
+    # Needed for CMake configurations for dependencies to work
+    generators = "CMakeDeps"
+
+    settings = "os", "compiler", "build_type", "arch"
```

Comment:
For 2.0 to be even smarter

```suggestion
    package_type = "library"
    settings = "os", "compiler", "build_type", "arch"
```